game/src/contrib/components/HealthComponent.java

```diff
@@ -21,41 +21,39 @@
 import java.util.logging.Logger;
 
 /**
- * The HealthComponent adds health points and the ability to take damage and die to an entity.
+ * Allow associated entity to take damage and die.
  *
- * <p>It keeps track of the current health points and its maximum.
+ * <p>The component keeps track of the current and maximum health points of the associated entity.
  *
- * <p>It also keeps track of the damage received via the {@link #receiveHit(Damage) receiveHit}
- * method. The damage is stored in a list and can be retrieved via the {@link
- * #calculateDamageOf(DamageType) getDamage} method. The damage is applied and cleared by the {@link
- * HealthSystem HealthSystem} every tick. To determine what the last cause of damage was, the {@link
- * #lastDamageCause()} method can be used.
+ * <p>The component also keeps track of the damage received via the {@link #receiveHit(Damage)
+ * receiveHit} method. The damage is stored in a list and can be retrieved via the {@link
+ * #calculateDamageOf(DamageType) calculateDamageOf} method. The damage is applied, and the list is
+ * cleared by the {@link HealthSystem HealthSystem} every frame.
```

Comment:
Bedeutet das, dass Schaden via receiveHit erstmal nur in der Liste zwischengespeichert wird und dann vom System "am Block" angewendet wird? Falls ja, wie ist die Interaktion des Systems (welche Methode aus der Component wird vom System dazu aufgerufen)? Falls nein, wieso kann ich dann nur den letzten Schaden abfragen?

Reply:
Hab die Klassenjavadoc erweitert